Formula/fish.rb

```diff
@@ -1,9 +1,16 @@
 class Fish < Formula
   desc "User-friendly command-line shell for UNIX-like operating systems"
   homepage "https://fishshell.com"
-  url "https://fishshell.com/files/2.3.1/fish-2.3.1.tar.gz"
-  mirror "https://github.com/fish-shell/fish-shell/releases/download/2.3.1/fish-2.3.1.tar.gz"
-  sha256 "328acad35d131c94118c1e187ff3689300ba757c4469c8cc1eaa994789b98664"
+
+  stable do
+    url "https://fishshell.com/files/2.3.1/fish-2.3.1.tar.gz"
+    mirror "https://github.com/fish-shell/fish-shell/releases/download/2.3.1/fish-2.3.1.tar.gz"
+    sha256 "328acad35d131c94118c1e187ff3689300ba757c4469c8cc1eaa994789b98664"
+
+    # Skip extra dirs creation during install phase patch for current stable. To be removed on the next fish release.
+    # As discussed in https://github.com/Homebrew/homebrew-core/pull/2813
+    patch :DATA
```

Comment:
Ah, I see it's done in https://github.com/fish-shell/fish-shell/commit/9abbc5f. You could use https://github.com/fish-shell/fish-shell/commit/9abbc5f.patch if that'll apply otherwise just note https://github.com/fish-shell/fish-shell/commit/9abbc5f in a comment above. Thanks!